server/views/case-summary-record-order.njk

```diff
@@ -21,6 +21,7 @@
 } %}
 
 {%- set communityData = data.communityData -%}
+{%- set breachData = data.communityData.breach %}
```

Comment:
Variable breachData has been created but unused

Reply:
delete it 👍